extensions/stats/plugin.cc

```diff
@@ -52,12 +52,11 @@ void PluginRootContext::onConfigure(std::unique_ptr<WasmData> configuration) {
     LOG_WARN("cannot parse local node metadata ");
     return;
   }
-  PluginDirection direction;
-  auto dirn_result = getPluginDirection(&direction);
-  if (WasmResult::Ok == dirn_result) {
-    outbound_ = PluginDirection::Outbound == direction;
+  int64_t direction;
+  if (getValue({"traffic_direction"}, &direction)) {
+    outbound_ = envoy::api::v2::core::TrafficDirection::OUTBOUND == direction;
```

Comment:
So PluginDirection is removed in envoy-wasm? This will make envoy proto a requirement for wasm compilation, not sure if it is possible to do it by 1.4...

Reply:
Proto enum is just an int. We can hardcode the value for OUTBOUND if you wish.

Reply:
In general, we may want to expose xDS to plugins that need to consume xDS config.
If config is available in xDS, we should not be using additional config on top of that to avoid proliferation of the config.